docs/installation/macm1.md

```diff
@@ -0,0 +1,34 @@
+# Installation guide for MAC with M1 chipset
```

Comment:
I guess most of this file will be obsolete once the images for Linux/arch64 will be built and pushed.

Reply:
ok I will delete it